### PR TITLE
[Debezium] Adding Geography + JSON converters

### PR DESCRIPTION
**Pull request by Tang8330**
Moving Geography and JSON to follow the converters pattern

### CODE REVIEW

File: lib/debezium/converters/geometry.go [hunk 1]
```diff
@@ -69,8 +69,14 @@ func (GeometryPoint) Convert(value any) (any, error) {
 	return string(bytes), nil
 }
 
-// ParseGeometry will take in an object with b64 encoded wkb and return a GeoJSON string.
-func ParseGeometry(value any) (string, error) {
+type Geography struct{}
```
Comment (danafallon) [inline lib/debezium/converters/geometry.go:72]: Is there a reason for calling this `Geography` instead of `Geometry` like it was before?

**Review by danafallon — APPROVED** [review]
One question about naming, otherwise looks good

File: lib/debezium/converters/geometry.go (hunk 1 above)
Comment (Tang8330) [reply lib/debezium/converters/geometry.go:72]: Haha yeah I named it Geography because we already have a variable named Geometry

Comment (Tang8330) [reply lib/debezium/converters/geometry.go:72]: And this parser handles both Geography + Geometry

Comment (Tang8330) [reply lib/debezium/converters/geometry.go:72]: I could rename [Geometry](https://github.com/artie-labs/transfer/blob/875987a24aa5e4248ba4704c39583c56203ad6b6/lib/debezium/converters/geometry.go#L23) to be GeometryJSON.

Wdyt?

Comment (danafallon) [reply lib/debezium/converters/geometry.go:72]: Ah okay, yeah I think that seems more consistent/user-friendly to me (Geometry -> GeometryJSON and Geography -> Geometry)

**Pull request merged**